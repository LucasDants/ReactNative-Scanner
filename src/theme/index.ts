export default {
  colors: {
    red: {
      900: '#B83341',
      800: '#E03F50',
    },
    blue: '#0000FF',
    white: '#FFFFFF',
    gray: {
      100: '#DCDCDC',
      600: '#404040',
      900: '#030303',
    },
  },
};