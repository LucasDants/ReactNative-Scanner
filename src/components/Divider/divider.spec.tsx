import React from 'react'

import { render } from '@testing-library/react-native'

import { ThemeProvider } from 'styled-components/native'
import theme from '../../theme'

import { Divider } from '.'

const Providers = ({ children }) => (
  <ThemeProvider theme={theme}>
    { children }
  </ThemeProvider>
)

describe('Divider Component', () => {

  it("should match the snapshot test", () => {
    const { toJSON } = render(
      <Divider />, {wrapper: Providers}
    )

    const component = toJSON()
    expect(component).toMatchSnapshot()

  })
})