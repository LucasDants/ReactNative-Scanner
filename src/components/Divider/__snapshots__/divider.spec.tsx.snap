// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider Component should match the snapshot test 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#404040",
        "height": 1,
        "width": "100%",
      },
    ]
  }
/>
`;
